lib/PEXv2.ts

```diff
@@ -275,7 +278,7 @@ export class PEXv2 {
    */
   public verifiablePresentationFrom(
     presentationDefinition: PresentationDefinitionV2,
-    selectedCredentials: OriginalVerifiableCredential[],
+    selectedCredentials: (IVerifiableCredential | JwtDecodedVerifiableCredential | CompactJWT)[],
```

Comment:
This is literally what OriginalVerifiableCredential is

Reply:
fixed